docker-compose.yaml

```
@@ -1,5 +1,155 @@
+            retries: 3
+            start_period: 10s
+            timeout: 10s
+        volumes:
+            - ./zoo-alert-manager/alertmanager.yml:/etc/alertmanager/config.yml
+            - alert-manager-data:/alertmanager
+        ports:
+            - "9093:9093"
+        command:
+            - --config.file=/etc/alertmanager/config.yml
+        depends_on:
+            - zoo-prometheus  
```

yamllint

[error] trailing spaces

(trailing-spaces)